arm-relay/2016-07-01/relay.json

```diff
@@ -1914,6 +2018,19 @@
     },
     "SharedAccessAuthorizationRuleResource": {
       "properties": {
+        "type": {
```

Comment:
How can you have a property named "type" if you are having an allOf on "Resource" which also has a property named type?